cmd/jujud/agent/machine.go

```diff
@@ -1254,6 +1258,72 @@ func (a *MachineAgent) newAPIserverWorker(
 	return server, nil
 }
 
+func getRateLimitConfig(cfg agent.Config) (apiserver.RateLimitConfig, error) {
```

Comment:
its a bit of a shame that you have to restart the agent to update these values, given that restarting a Controller means killing and restarting 1000 active connections.
"juju model-config -m controller" has the *super* nice property that the changes are detected live.
(it often takes 30min for a controller to come anywhere close to 'settled' after bouncing.)